fastlane/Fastfile

```diff
@@ -638,16 +689,23 @@ REPOSITORY_NAME="WordPress-Android"
   #####################################################################################
   desc "Builds an app bundle"
   lane :build_for_translation_review do | options |
+
+  translator_locales = %w[de es fr id it ja ko nl ru sv zh-cn zh-tw tr he pt ar pl th sr vi]
+  translator_locales_map = ALL_LOCALES.select { translator_locales.include?(h[:glotpress]) }
+
+    android_download_translations(
+      glotpress_url: 'https://translate.wordpress.com/projects/wporg/apps/android',
+      status_filter: ['current', 'waiting', 'fuzzy'],
+      locales: translator_locales_map,
+      lint_task: nil, skip_commit: true
+    )
+
     cmd_params = ""
     unless options[:custom_version].nil?
       UI.message("Building custom version: #{options[:custom_version]}")
       cmd_params=" -PversionName=\"#{options[:custom_version]}\""
     end
 
-    sh("../tools/update-translations.sh review")
-    sh("../tools/update-translations.sh review waiting")
-    sh("../tools/update-translations.sh review fuzzy")
-
     android_merge_translators_strings(strings_folder:"./WordPress/src/main/res/")
```

Comment:
This step should be removed too as the merging is already handled by `android_download_translations` when passing multiple values (here, 3) for `status_filter`.

```suggestion
```